idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -677,6 +688,10 @@ def rule_mass_transfer_coeff_vap(blk, t, x, j):
                       blk.vapor_phase.properties[t, x].dens_mass) /
                      (blk.a_ref *
                       blk.vapor_phase.properties[t, x].visc_d))**(3/4))
+            elif x == self.vapor_phase.length_domain.first():
+                return Expression.Skip
+            else:
+                return Expression.Skip
 
         self.k_v = Expression(self.flowsheet().time,
                               self.vapor_phase.length_domain,
```

Comment:
Further down we have a CO2 specific term - we will need to clean that up too (GitHub won't let me put a comment on the actual line as it wasn't edited).